GothicVRProject/GothicVR
# Feature/npc sword

Please review and merge after https://github.com/GothicVRProject/GothicVR/pull/169

# To test
1. Load world.
2. Check if OC NPCs have swords (1handed or 2handed)
3. ...


# Checklist
(Please judge for yourself which entry is valuable for your PR)

## Scene

# Testing
